python/arcticdb/version_store/_normalization.py

```diff
@@ -301,11 +293,21 @@ def _normalize_single_index(index, index_names, index_norm, dynamic_strings=None
             index_tz = get_timezone(index[0].tzinfo)
 
         if index_tz is not None:
-            index_norm.tz = index_tz
+            index_norm.tz = _ensure_str_timezone(index_tz)
 
         return index_names, ix_vals
 
 
+def _ensure_str_timezone(index_tz):
+    if is_utc(index_tz):
```

Comment:
Unit tests failing here as `index_tz` is a `str` here for versions of `pandas` where `get_timezone` **does** still return a `str` for `UTC`.

```suggestion
    if not isinstance(index_tz, str) and is_utc(index_tz):
```

Reply:
Updated with a check that `index_tz` is a `tzinfo` object to match Panda's `is_utc` strict typing.